tests/e2e/specs/modules/search-console/dashboard-date-range.test.js

```diff
@@ -36,7 +36,16 @@ import * as dashboardRequests from './fixtures/dashboard';
 import * as dashboardDetailsRequests from './fixtures/dashboard-details';
 import * as modulePageRequests from './fixtures/module-page';
 
+// TODO The dashboard and page dashboard tests still use legacy widgets
+// and require legacy fixtures via calls to /data. Once they are refactored, this
+// can be removed and the request interception can be brought in line with the other
+// Widget API tests (see below).
 let mockBatchResponse;
+// TODO The module page tests here now use the Widget API. They no longer call
+// /data and receive batched responses. To make the distinction clear, these tests
+// make use of this variable instead of `mockBatchResponse`. As part of #2586,
+// this can be refactored to use the new getSearchConsoleMockResponse utility.
+let mockResponse;
```

Comment:
```suggestion
// TODO The module pages use the Widget API. They no longer call
// /data and receive batched responses. To make the distinction clear, these tests
// make use of this variable instead of `mockBatchResponse`. As part of #2586,
// this can be refactored to use the new getSearchConsoleMockResponse utility.
let mockResponse;
```